Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "adblock-ffi"
+version = "0.1.0"
+authors = ["Brian R. Bondy <netzen@gmail.com>"]
+edition = "2018"
+
+[dependencies]
+adblock="0.1.0"
+
+[lib]
+crate-type = [
+    "cdylib",
+    "staticlib",
+]
+name = "adblock"
+
+[patch.crates-io]
+adblock = { path = "../adblock-rust" }
+
+[features]
+cbindgen = []
```

Comment:
I think you need to set the panic behavior to abort, unwind is undefined behavior across the FFI boundry

Reply:
Thanks, added.

Just this right?

```
[profile.dev]
panic = "abort"

[profile.release]
panic = "abort"
```